otx/cli/utils/parser.py

```diff
@@ -143,6 +143,11 @@ def _get_leaf_node(curr_dict: Dict[str, dict], curr_key: str):
         value_type = None
         if type_hint is not None:
             value_type = type_hint.get(origin_key, {}).get("type", None)
+        # FIXME[HARIM]: There's no template in args, and it's not inside the workspace, but with --workspace,
+        # the template is not found in args, so params, which are all bools, go into str.
+        # This is a temporary solution.
+        if isinstance(value, str) and value.lower() in ("true", "false"):
+            value_type = str2bool
```

Comment:
Is is okay to use? 

I think the thing below is correct
```python3
value_type = str2bool(value)
```

Reply:
This is used on line 153
`leaf_node_dict[node_key] = {"value": value_type(value) if value_type else value}`
, so the value_type itself is expecting a function or type.

Reply:
Got it, quite interesting usage to me.